UPDATING.md

```diff
@@ -594,8 +594,8 @@ log_filename_template = {{{{ ti.dag_id }}}}/{{{{ ti.task_id }}}}/{{{{ ts }}}}/{{
 log_processor_filename_template = {{{{ filename }}}}.log
 
 [elasticsearch]
-elasticsearch_log_id_template = {{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}
```

Comment:
We should leave these as they are as they were right for the time.

We should add a new section at the top under "master" about the rename.

Reply:
@ashb done.